consensus/misc/kip71_test.go

```diff
@@ -20,6 +20,80 @@ func getTestConfig(forkedBlockNum *big.Int) *params.ChainConfig {
 	return testConfig
 }
 
+func TestEvenBaseFee(t *testing.T) {
+	tests := []struct {
+		upperBoundBaseFee         uint64
+		lowerBoundBaseFee         uint64
+		gasTarget                 uint64
+		baseFeeDenominator        uint64
+		maxBlockGasUsedForBaseFee uint64
+		parentGasUsed             uint64
+		parentBaseFee             uint64
+	}{
+		// Current default setting
+		{750000000000, 25000000000, 30000000, 20, 60000000, 43212345, 13009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 34857284, 83009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 0, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 0, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 0, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 69856837, 34958439443},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 18275847, 43459029443},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 34857359, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 28914728, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 83597392, 43238573843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 28472874, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 34895734, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 17274858, 18432574843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 39093494, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 18981884, 28574384343},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 12873, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 18273, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 1, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 18949, 43009212843},
+		{750000000000, 25000000000, 30000000, 20, 60000000, 19828, 43009212843},
+		// monkey test 1
+		{750000000001, 25123987327, 30000000, 20, 38573282, 83212345, 750000000000},
```

Comment:
Can we add one more case that tests with odd number lower bound base fee and parents gas used 0 (next base fee is lower bound base fee + 1)

Reply:
Sure, I added {750000000000, 1, 30000000, 1, 60000000, 0, 10} and logic checking lower/upper bound

Reply:
Thank you for adding that! But can you check test code comments? The other cases are commented.